app/controllers/registrations_controller.rb

```diff
@@ -822,19 +823,69 @@ def create
   end
 
   def create_paypal_order
-    return head :forbidden if Rails.env.production?
+    return head :forbidden if PaypalInterface.paypal_disabled?
 
     @registration = registration_from_params
-    render json: PaypalInterface.create_order(@registration)
+    render json: PaypalInterface.create_order(@registration, params[:total_charge])
   end
 
   def capture_paypal_payment
-    return head :forbidden if Rails.env.production?
+    return head :forbidden if PaypalInterface.paypal_disabled?
 
     @registration = registration_from_params
     @competition = @registration.competition
     order_id = params[:order_id]
 
-    render json: PaypalInterface.capture_payment(@competition, order_id)
+    response = PaypalInterface.capture_payment(@competition, order_id)
+    if response['status'] == 'COMPLETED'
+
+      # TODO: Handle the case where there are multiple captures for a payment
+      # 1) Multiple installments
+      # 2) Some failed, some succeeded
+
+      amount_details = response['purchase_units'][0]['payments']['captures'][0]['amount']
+      currency_code = amount_details['currency_code']
+      amount = PaypalRecord.amount_in_cents(amount_details["value"], currency_code)
```

Comment:
Nit: Double-quotes used here unnecessarily. We may want to change overcommit so that it picks this up? 

@gregorbg again happy to make a change now or in a subsequent PR, up to you

Reply:
I am genuinely surprised that it doesn't pick this up already.

If we are to make this change, it should be a separate PR. That separate PR should then also rectify all other "mistakes" throughout the entire codebase (which are probably a lot!)

Reply:
11000 instances, I'll raise the PR :+1: